src/test/groovy/org/cyclonedx/gradle/DependencyResolutionSpec.groovy

```diff
@@ -411,6 +411,27 @@ class DependencyResolutionSpec extends Specification {
         javaVersion = TestUtils.javaVersion
     }
 
+    def "no url in distributionManagement repository"() {
+        given:
+        File testDir = TestUtils.duplicate("bad-distributionManagement")
```

Comment:
Since it's a simple test project it'd be better to declare it fully in the test case like here https://github.com/CycloneDX/cyclonedx-gradle-plugin/pull/673/files#diff-f1d9df05332d3202c88f3d7e8eed67baf92bfb2aced129f9db6ab51bf08b7a56R231-R246

Reply:
Done